test/jest-api.setup.js

```diff
@@ -1,8 +1,11 @@
 'use strict';
 
 const isoDateRegex = /\d{4}-\d{2}-\d{2}T\d{2}:\d{2}:\d{2}.\d{3}Z/;
-
-jest.setTimeout(60000);
+process.env.DATABASE_NAME = `strapi_${process.env.JEST_WORKER_ID}`;
+if (process.env.DATABASE === 'mysql') {
+  process.env.DATABASE_USERNAME = 'root';
+}
```

Comment:
This feels hacky and not right to be here . I wouldn't get why we are changing this here by reading this.

Reply:
Let me change this to a variable you put into the the function I guess. I agee this is to hacky